### PR TITLE
[crunchyroll] Extract season name as well as the series

### PR DESCRIPTION
**Pull request by Starsam80**
### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/rg3/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/rg3/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [ ] Bug fix
- [x] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

In addition to extracting the `series`, we should also extract the `season` name as well. See the test I added for an example.

### CODE REVIEW

File: youtube_dl/extractor/crunchyroll.py [hunk 1]
```diff
@@ -492,6 +510,7 @@ def _real_extract(self, url):
         series = self._html_search_regex(
             r'id=["\']showmedia_about_episode_num[^>]+>\s*<a[^>]+>([^<]+)',
             webpage, 'series', default=xpath_text(metadata, 'series_title'))
```
Comment (dstftw) [inline youtube_dl/extractor/crunchyroll.py]: This fallback should be removed now.

Comment (Starsam80) [inline youtube_dl/extractor/crunchyroll.py]: So `default=None`?

Comment (dstftw) [inline youtube_dl/extractor/crunchyroll.py]: `fatal=False`.

**Comment by Starsam80** [comment]
Hopefully this is what you wanted

**Pull request merged**